### ISSUE TITLE
I don't see the Razor View in the list anymore. I believe it might have been changed to MVC View Page

### ISSUE THREAD

**Issue by Prakash94** [aspnet/Docs#6569]
In the Add New Item - MvcMovie dialog

In the search box in the upper-right, enter view

Tap Razor View

In the Name box, change the name if necessary to Index.cshtml.

Tap Add

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 788d46c9-a91a-2788-8db7-b53c37b512f2
* Version Independent ID: 00c2c01d-d235-2d2d-5c96-1f33a9314382
* Content: [Adding a view](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-view?view=aspnetcore-2.0#feedback)
* Content Source: [aspnetcore/tutorials/first-mvc-app/adding-view.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/first-mvc-app/adding-view.md)
* Service: **unspecified**
* Product: **asp.net-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
Can you supply the exact text that is wrong or supply a screen shot? I don't see what's wrong.

**Comment by Prakash94** [comment]
![image](https://user-images.githubusercontent.com/12123332/40522505-d61f3122-6003-11e8-806b-9cffac83ce3e.png)

Yes here it is

**Comment by Rick-Anderson** [comment]
What version of Visual Studio are you using and what version of .NET Core do you select? I see it
![image](https://user-images.githubusercontent.com/3605364/40524218-14b0ed7a-5f75-11e8-82ad-d78476cae6bb.png)

**Comment by Rick-Anderson** [comment]
What exact version?
![image](https://user-images.githubusercontent.com/3605364/40525468-eda7bce8-5f7b-11e8-8fdb-d70e4e70e51b.png)

What do you see?

**Comment by Rick-Anderson** [comment]
Did you install the web development workload

**Comment by Rick-Anderson** [comment]
And why are you doing MVC and not Razor Pages?

**Comment by Prakash94** [comment]
I see this:

![image](https://user-images.githubusercontent.com/12123332/40526311-4cd739b8-6018-11e8-9fcf-64aac7fd1d0f.png)

Yes I installed the web development workload.

Oh I did MVC because I couldn't find the Razor View. The description was the same so i thought it would be similar. I needed to learn it quick and only up to the view part.

I can see my knowledge is a bit lacking on this area. I thought others might face the same issue hence why I commented. Anyway, thank  you for your time Rick, and really sorry for the bother. I will read up on Razor Pages, MVC, etc.
